src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,103 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Use the diamond operator to ensure type safety. It should be `new Node<K,V>[DEFAULT_INITIAL_CAPACITY]`. Be aware that you cannot create an array of a generic type directly, so consider using an unchecked cast with a SuppressWarnings annotation or an alternative approach like an ArrayList.